Cargo.toml

```diff
@@ -52,6 +52,13 @@ unused_unsafe = "warn"
 missing_unsafe_on_extern = "warn"
 unsafe_attr_outside_unsafe = "warn"
 
+# == Correctness == #
+ambiguous_negative_literals  = "warn"
+arithmetic_overflow = "warn"
```

Comment:
This is warn-by-default, so we can remove

Reply:
Removed